library/core/src/iter/adapters/array_chunks.rs

```diff
@@ -71,76 +71,3 @@ impl<I: Iterator, const N: usize> Iterator for ArrayChunks<I, N> {
 
 #[unstable(feature = "iter_array_chunks", issue = "none")]
 impl<I: FusedIterator, const N: usize> FusedIterator for ArrayChunks<I, N> {}
-
-/// An iterator that yields the elements of another iterator in
-/// chunks of size `N` starting from the end.
-///
-/// This `struct` is created by the [`array_rchunks`] method on [`Iterator`]. See
-/// its documentation for more.
-///
-/// [`array_rchunks`]: Iterator::array_rchunks
-#[unstable(feature = "iter_array_chunks", issue = "none")]
-#[derive(Debug)]
-pub struct ArrayRChunks<I: DoubleEndedIterator, const N: usize> {
```

Comment:
Is `array_rchunks` actually redundant? Seems to me it's distinct from both `.rev().array_chunks()` (because the elements within the chunks are in the original order) and `.array_chunks().rev()` (when `.remainder().len() != 0`).

Reply:
It's mostly redundant with the exception of the remainder its equivalent to:
`.rev().array_chunks().map(|mut arr| { arr.reverse(); arr})`

If it still seems useful, I can re-include it.

Reply:
I have now added a `DoubleEndedIterator` implementation that seems to produce the same results `array_rchunks` would. It's not quite as efficient as it could be though, just to keep complexity down and share more code with the forward implementation.